snapcraft/tests/unit/project_loader/test_config.py

```diff
@@ -1916,9 +1893,11 @@ def setUp(self):
             'version': '1.0-snapcraft1~ppa1',
             'summary': 'my summary less that 79 chars',
             'description': 'description which can be pretty long',
+            'adopt-info': 'part1',
```

Comment:
why is this necessary?

Reply:
Just so I don't have to explicitly add it in the test cases. Harmless for the other ones.